app/views/step_by_step_pages/schedule.html.erb

```diff
@@ -0,0 +1,82 @@
+<%
+  links = [
+    {
+      text: 'Step by step publisher',
+      href: step_by_step_pages_path
+    },
+    {
+      text: @step_by_step_page.title,
+      href: step_by_step_page_path(@step_by_step_page)
+    },
+    {
+      text: 'Schedule'
+    }
+  ]
+%>
+
+<%= render 'shared/steps/step_breadcrumb', links: links %>
+
+<%= render 'shared/steps/heading',
+  step_nav: @step_by_step_page.title,
+  action: 'Set a date and time to publish'
+%>
+
+<%= render 'nav', step_by_step_page: @step_by_step_page, active: "schedule" %>
+
+<% if @schedule_intent %>
+  <%= render 'shared/steps/form_errors', resource: @schedule_intent %>
+<% end %>
+
+<% if @step_by_step_page.status[:name] == 'live' %>
+  <div class="callout callout-warning add-bottom-margin">
+    <div class="callout-title">
+      Warning
+    </div>
+    <div class="callout-body">
+      There are no changes to schedule!
+    </div>
+  </div>
+<% elsif @step_by_step_page.scheduled_for_publishing? %>
+  <div class="alert alert-info">
+    Scheduled to be published at <%= @step_by_step_page.scheduled_at %>
+  </div>
+<% else %>
+  <div class="row">
+    <div class="col-md-7">
+      <%= form_tag do %>
+        <div class="form-group">
+          <%= label_tag :scheduled_at, 'Schedule datetime (e.g. "2020-04-20 10:26:51 UTC")' %>
+          <%= text_field_tag :scheduled_at, params[:scheduled_at]  %>
+        </div>
+
+        <div class="form-group">
+          <%= label_tag :change_note, "Change note" %>
+          <%= text_field_tag :change_note, params[:change_note]  %>
+        </div>
+
+        <div class="form-group">
+          <%= label_tag :update_type, "Update type" %>
```

Comment:
Same here. If we assume that the scheduled publication is a minor change, we can remove this check.